.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: node_js
+node_js:
+  - '8.1.3'
```

Comment:
Hint: You can just write `node` here to keep the latest LTS node version.
Or just write `8` to heep the latest 8 version.